images/build/Dockerfiles/Dockerfile

```diff
@@ -79,24 +79,24 @@ RUN set -ex \
  # 1.1 sdk <-- 1.0 runtime's sdk
  && mkdir $NET_CORE_APP_10 \
  && ln -s $NET_CORE_APP_10 1.0 \
- && ln -s $sdksDir/$DOT_NET_CORE_11_SDK_VERSION $NET_CORE_APP_10/sdk \
+ && echo $DOT_NET_CORE_11_SDK_VERSION > $NET_CORE_APP_10/sdkVersion.txt \
  # 1.1 sdk <-- 1.1 runtime's sdk
  && mkdir $NET_CORE_APP_11 \
  && ln -s $NET_CORE_APP_11 1.1 \
  && ln -s 1.1 1 \
- && ln -s $sdksDir/$DOT_NET_CORE_11_SDK_VERSION $NET_CORE_APP_11/sdk \
```

Comment:
opting to use a file to store metadata rather than a link because there are cases where C# code needs to get info of the sdk based on runtime version and it is not easy to read a link from c# currently.